components/financial_assistance/app/views/financial_assistance/applicants/_dependent_applicant_income_and_coverage_info.html.erb

```diff
@@ -0,0 +1,38 @@
+<div id="add_member_list_<%= dependent.id %>" class="dependent_list py-5">
```

Comment:
the py-5 seems pretty high and each member has a ton of space around it - not sure if that is just following mockups